src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+import java.io.*;
+import java.util.*;
+import java.util.Date;
 public class Duke {
+    static String line = "    ____________________________________________________________";
+    public static LinkedList<Task> storage = new LinkedList<Task>(); 
     public static void main(String[] args) {
+        
+        Scanner sc = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println(line);
+
+        int count = 0;
+        int taskaddnum = 0;
+        while(true){
+            String inputstring = sc.nextLine();
+            if(inputstring.equals("bye")){
+                // END //
+                output("Bye. Hope to see you again soon!");
+                break;
+            }
+            else if(inputstring.equals("list")){
+                //   SHOW LIST //
+                displaycurrentlist();
+            }
+            else if(inputstring.split(" ")[0].equals("done")){
+                // MARKIN TASK DONE //
+                //System.out.println(Integer.parseInt(inputstring.split(" ")[1]));
+                try{
+                    markdone(Integer.parseInt(inputstring.split(" ")[1]));
+                }
+                catch(Exception e){
+                    output("☹ OOPS!!! The description of a done cannot be empty.");
+                }
+            }
+            else if(inputstring.split(" ")[0].equals("delete")) {
+                try{
+                    delete(Integer.parseInt(inputstring.split(" ")[1]));
+                }
+                catch(Exception e){
+                    output("☹ OOPS!!! The description of a delete cannot be empty.");
+                }
+            }
+            else if(inputstring.split(" ")[0].equals("todo")){
+                // ADD TASK TO LIST //
+                try{
+                    System.out.println(inputstring.substring(5));
+                    add(taskaddnum+1, 
+                        inputstring.substring(5),
+                        false,
+                        "ToDO",
+                        "__",
+                        "__");
+                    taskaddnum ++;
+                }
```

Comment:
i like how you add single comments in each block, it makes the code easy to read and understand.